packages/odyssey-react/src/components/FieldGroup/index.tsx

```diff
@@ -10,71 +10,4 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import React from "react";
-import type {
-  ComponentProps,
-  FunctionComponent,
-  ReactNode,
-  ReactElement,
-} from "react";
-import { useOmit, withStyles } from "../../utils";
-
-import styles from "./FieldGroup.module.scss";
-
-interface Props {
-  /**
-   * Content to be rendered within the FieldGroup.
-   */
-  children: ReactElement | ReactElement[];
-
-  /**
-   * The title of the FieldGroup.
-   */
-  title?: string;
-
-  /**
-   * A short description of the FieldGroup.
-   */
-  desc?: string;
-}
-
-interface PropsError {
-  children: ReactNode;
-}
-
-type Statics = {
-  Error: typeof Error;
-};
-
-let FieldGroup: FunctionComponent<Props> & Statics = Object.assign(
-  (props: Props) => {
-    const { children, title, desc, ...rest } = props;
-
-    const omitProps = useOmit(rest);
-
-    return (
-      <fieldset {...omitProps} className={styles.root}>
-        {title && <legend className={styles.title}>{title}</legend>}
-        {desc && <p>{desc}</p>}
-        {children}
-      </fieldset>
-    );
-  },
-  {
-    Error,
-  }
-);
-
-function Error({ children }: PropsError) {
-  return <section className={styles.error}>{children}</section>;
-}
-
-FieldGroup.displayName = "FieldGroup";
-Error.displayName = "FieldGroupError";
-
-FieldGroup.Error = Error;
-
-FieldGroup = withStyles(styles)(FieldGroup);
-
-export type FieldGroupProps = ComponentProps<typeof FieldGroup>;
-export { FieldGroup };
+export * from "./FieldGroup";
```

Comment:
thank you for creating a barrel file. It does not need a `.tsx` extension however you can use a plain `.ts` here